gdnative-core/src/nativescript/macros.rs

```diff
@@ -57,6 +57,16 @@ macro_rules! godot_wrap_method_parameter_count {
     }
 }
 
+#[doc(hidden)]
+#[macro_export]
+macro_rules! godot_wrap_method_inner_strip_ref {
+    (& $lt:lifetime mut $(rest:tt)*) => ($($rest)*);
+    (& $lt:lifetime $(rest:tt)*) => ($($rest)*);
+    (& mut $(rest:tt)*) => ($($rest)*);
+    (& $(rest:tt)*) => ($($rest)*);
+    ($(rest:tt)*) => ($($rest)*);
```

Comment:
Could the `?` modifier be used here? Something like `& $lt:lifetime? mut? $(rest:tt)*`?

Reply:
This macro was actually unused! Removed it.